assets/app/view/game/actionable.rb

```diff
@@ -95,6 +97,30 @@ def clear_ui_state
         store(:selected_company, nil, skip: true)
         store(:tile_selector, nil, skip: true)
       end
+
+      def history_link(text, title, action_id = nil, style_extra = {})
```

Comment:
why do you need sytle extra seems unnecessary

Reply:
Because the links in the history control (where this is being generalised from) have a margin in the anchor tag that shouldn't apply to links else. I can wrap them in a span with the margin instead, if you prefer.